File: src/main/java/my/maryvk/maryvkweb/service/UserService.java
package my.maryvk.maryvkweb.service;

import my.maryvk.maryvkweb.domain.User;

import java.util.List;

public interface UserService {
    boolean exists(Integer id);
    User findOne(Integer id);
    void save(Iterable<User> users);
    void save(User user);
}

File: src/main/java/my/maryvk/maryvkweb/service/UserService.kt
package my.maryvk.maryvkweb.service

import my.maryvk.maryvkweb.domain.User

interface UserService {
    fun exists(id: Int): Boolean
    fun findOne(id: Int): User
    fun save(users: Iterable<User>)
    fun save(user: User)
}